Bottom: 809aec71c8c051820776057931e66d073bec580e
Top:    3c68ae697f7c4e15e7346c72d61767c3b43fb14c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-11 17:02:17 -0500

Refresh of update-repo-forks

---
diff --git a/ansible-playbook/group_vars/docker-hosts/repos.yml b/ansible-playbook/group_vars/docker-hosts/repos.yml
new file mode 100644
index 00000000..cc4d9cbf
--- /dev/null
+++ b/ansible-playbook/group_vars/docker-hosts/repos.yml
@@ -0,0 +1,13 @@
+---
+
+buildbot_image_repo: https://github.com/AndrewCooper/docker-buildbot-master.git
+buildbot_worker_ntel_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ntel.git
+buildbot_worker_ptxdist_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ptxdist.git
+exim4_image_repo: https://github.com/AndrewCooper/docker-exim4.git
+gitlab_image_repo: 'https://github.com/AndrewCooper/docker-gitlab.git'
+mantisbt_image_repo: https://github.com/AndrewCooper/docker-mantisbt.git
+openldap_image_repo: https://github.com/AndrewCooper/docker-openldap.git
+openssl_image_repo: https://github.com/AndrewCooper/docker-openssl.git
+phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
+svn_image_repo: https://github.com/AndrewCooper/docker-svn.git
+wiki_image_repo: https://github.com/AndrewCooper/docker-mediawiki.git
